input/blog/2020-11-05-cake-v1.0.0-rc0001-released.md

```diff
@@ -0,0 +1,234 @@
+We're now utilizing [Spectre.Cli](https://www.nuget.org/packages/Spectre.Cli/) for a more robust, flexible and future prof handling of
+
+Full details of everything that was included in this release can be seen below.
+
+### Frosting incorporated into main rep
+
+Cake Frosting is now incorporated in same GitHub repository as ( [cake-build/cake](https://github.com/cake-build/cake/) ) all Cake other Cake runners, this means it'll be released and supported with the same cadence as them.
+
+### Supported platforms
+
+| Runner | .NET 5 | .NET 4.6.1 | .NET Core 3.1 | .NET Core 3.0 | .NET Core 2.1 | .NET Core 2.0 | .NET Standard 2.0 | Mono |
+|--------|--------|------------|---------------|---------------|---------------|---------------|-------------------|------|
+| [Cake.Tool](https://www.nuget.org/packages/Cake.Tool/1.0.0-rc0001)           | ✔  | ❌ | ✔  | ✔  | ✔  | ❌ | ❌ | ❌ |
+| [Cake.Frosting](https://www.nuget.org/packages/Cake.Frosting/1.0.0-rc0001)   | ✔  | ✔  | ✔  | ✔  | ✔  | ✔  | ✔  | ✔  |
+| [Cake](https://www.nuget.org/packages/Cake/1.0.0-rc0001)                     | ❌ | ✔  | ❌ | ❌ | ❌ | ❌ | ❌ | ✔  |
+| [Cake.CoreCLR](https://www.nuget.org/packages/Cake.CoreCLR/1.0.0-rc0001)     | ❌ | ❌ | ❌ | ❌ | ❌  | ✔ | ❌ | ❌ |
+
+### Supported operating systems
+
+| Runner | macOS | Linux | Windows |
+| ------ | ----- | ----- | ------- |
+| [Cake.Tool](https://www.nuget.org/packages/Cake.Tool/1.0.0-rc0001)           | ✔  | ✔  | ✔  |
+| [Cake.Frosting](https://www.nuget.org/packages/Cake.Frosting/1.0.0-rc0001)   | ✔  | ✔  | ✔  |
+| [Cake](https://www.nuget.org/packages/Cake/1.0.0-rc0001)                     | ✔  | ✔  | ✔  |
+| [Cake.CoreCLR](https://www.nuget.org/packages/Cake.CoreCLR/1.0.0-rc0001)     | ✔  | ✔  | ✔  |
+
+### Supported CI Systems
+
+| Runner | AppVeyor | Azure Pipelines | Bamboo | Bitbucket Pipelines | Bitrise | Continua CI | GitHub Actions | GitLab CI | GoCD | Jenkins | MyGet | TeamCity | TravisCI |
+| - | - | - | - | - | - | - | - | - | - | - | - | - | - |
+| [Cake.Tool](https://www.nuget.org/packages/Cake.Tool/1.0.0-rc0001)           | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
+| [Cake.Frosting](https://www.nuget.org/packages/Cake.Frosting/1.0.0-rc0001)   | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
+| [Cake](https://www.nuget.org/packages/Cake/1.0.0-rc0001)                     | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
+| [Cake.CoreCLR](https://www.nuget.org/packages/Cake.CoreCLR/1.0.0-rc0001)     | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
+
+### How can you help?
+
+Help us by taking Cake 1.0.0 RC 1 for spin, test your scripts, addins and modules, if you have have questions please raise them on our [GitHub Discussions](https://github.com/cake-build/cake/discussions) and report any issues on our [GitHub Issues](https://github.com/cake-build/cake/issues).
+
+Easiest and safest way to test is by using [Cake.Tool](https://www.nuget.org/packages/Cake.Tool/1.0.0-rc0001) with a local tool manifest.
```

Comment:
Link to https://cakebuild.net/docs/running-builds/runners/dotnet-core-tool instead of nuget.org page

Reply:
adjusted